### PR TITLE
Downloads Panel - does not show if there are no downloadableTypes available

### PR DESCRIPTION
**Pull request by iisa**
+ unit test
+ demo

### Testing https://deploy-preview-1172--lucid-poitras-9a1249.netlify.app/bookreaderdemo/demo-internetarchive?ocaid=theworksofplato01platiala
- go to review app 
- open side panel to confirm download pane is there
- close side panel
- click - Remove Download panel
- open side panel
- ✅ no download panel anymore
  - image

### CODE REVIEW

**Review by dualcnhq — APPROVED** [review]
:lgtm:

**Review by cdrini — CHANGES_REQUESTED** [review]
Lgtm! I think we should remove the option to hide the download options from the demo; I'm not sure what value it will provide for the demo going forward outside the context of this PR. Our demos are getting a little large and difficult to maintain cause they're written in older JS. Keeping it slim is preferable :)

File: BookReaderDemo/IADemoBr.js [hunk 1]
```diff
@@ -91,6 +91,17 @@ const initializeBookReader = (brManifest) => {
...
+downloadPanel.addEventListener('click', async () => {
+  const iaBr = document.querySelector('ia-bookreader');
+  const bookNav = iaBr.shadowRoot.querySelector('book-navigator');
+
+  bookNav.downloadableTypes = [];
+
+  bookNav.updateMenuContents();
+  await bookNav.updateComplete;
+});
+
```
[the remark this answers is not in the document]
Comment (iisa) [reply BookReaderDemo/IADemoBr.js:104]: ok sound good. i can remove.

**Review by cdrini — APPROVED** [review]
Lgtm!

**Pull request merged**